integration/Jenkinsfiles/shared.groovy

```diff
@@ -0,0 +1,59 @@
+def warningsPublisher() {
```

Comment:
header required
method documentation is required, usage samples

Reply:
maybe worth to introduce a marker or a TODO comment for methods candidate to being moved to the Nuxeo Pipeline Library
`// TODO canditate for nuxeo-pipeline-library`